source/components/recipe-page.js

```diff
@@ -114,7 +116,7 @@ export class RecipePage extends HTMLElement {
 
     shadow.getElementById("recipe-image").src = recipeObj.image;
     shadow.getElementById("recipe-description").innerHTML =
-      recipeObj.description;
+      recipeObj.description.replace(ANCHOR_REGEX, ""); // remove links
```

Comment:
We should also store this new description in the recipe page's recipe object, this is to make the removed link tags not show up when editing the recipe description
```
      let modifiedDescription = recipeObj.description.replace(ANCHOR_REGEX, ""); // remove links
      shadow.getElementById("recipe-description").innerHTML = modifiedDescription;
      this.recipe.description = modifiedDescription;
```